anni-provider/Cargo.toml

```diff
@@ -33,3 +33,12 @@ anni-flac = { path = "../anni-flac", version = "0.2.2", features = ["async"] }
 version = "0.11"
 default-features = false
 features = ["json", "rustls-tls", "stream"]
+optional = true
+
+[features]
+default = ["convention", "drive", "proxy", "strict"]
```

Comment:
add `full` feature and use it as default?

Reply:
added